static/js/apps/eval_rig/util.ts

```diff
@@ -16,6 +16,8 @@
 
 const httpPattern = /https:\/\/[^\s]+/g;
 
+const longSpaces = "&nbsp;&nbsp;&nbsp;&nbsp;";
```

Comment:
I see that long spaces used to be 2 spaces, did you mean to update to 4 spaces?

Reply:
Yes, purposely making this larger so it's not skipped.